module/VuFind/src/VuFind/Db/Row/Resource.php

```diff
@@ -220,6 +220,8 @@ public function addOrUpdateRating(int $userId, ?int $rating): int
      * @param \VuFind\Date\Converter            $converter Date converter
      *
      * @return \VuFind\Db\Row\Resource
+     *
+     * @deprecated Use \VuFind\Record\ResourcePopulator::assignMetadata()
```

Comment:
There's not an easy way to make the old code a wrapper around the new code, so I'm just deprecating it as-is.